products/confluence.md

```diff
@@ -12,7 +12,18 @@ eolColumn: Support
 releaseColumn: true
 releaseDateColumn: true
 
+# release date: https://www.atlassian.com/software/confluence/download-archives
+# eol date: https://confluence.atlassian.com/support/atlassian-support-end-of-life-policy-201851003.html
 releases:
+-   releaseCycle: "8.6"
+    releaseLabel: "8.6 Data Center"
```

Comment:
Would make much sense to document this in the description for now IMO. Will do it.

Reply:
I noticed the special label on https://www.atlassian.com/software/confluence/download-archives and thought to show the detail also in the cycle here. I am okay with the warn you added in favour of the label :+1: